people.html

```diff
@@ -437,7 +440,7 @@ <h1 id="page-title" ng-non-bindable="">GEOS-Chem People and Projects </h1>
   L.marker([34.201467774645586, -118.17093305334618], { icon: greenIcon }).addTo(mymap).bindPopup("<a href='#jpl'><strong>NASA Jet Propulsion Laboratory<br>Pasadena, California, USA</strong></a>");
   
   // NASA Langley Research Center Hampton, Virginia, USA
-  L.marker([37.096175668285525, -76.38332286758234], { icon: greenIcon }).addTo(mymap).bindPopup("<a href='#langley'><strong>NASA Langley Research Center<br>Hampton, Virginia, USA</strong></a>");
+  L.marker([37.096175668285525, -76.38332286758234], { icon: greenIcon }).addTo(mymap).bindPopup("<a href='#langley'><strong>NASA Langley Research Center<br>Hampton, Virginia, USA</strong></a><ul><li>Research Group of T. Duncan Fairlie</li><li>Research Group of Katherine Travis</li></ul>");
```

Comment:
@SaptSinha Could you please remove the line for Duncan. He is no longer with us.